hal/lp_ticker_api.h

```diff
@@ -49,37 +49,126 @@ void lp_ticker_irq_handler(void);
 
 /** Initialize the low power ticker
  *
+ * Pseudo Code:
```

Comment:
Could you detail what initialization mean  ? 
* Does it start the ticker ? 
* Does it enable the interrupt ? 
* What should be the behavior if the ticker was already initialized ? 

There is more details in the `us_ticker_init` documentation.

Reply:
done